tests/unit/utils/guildRole.test.ts

```diff
@@ -129,3 +132,71 @@ describe("removeGuildRole", () => {
     expect(result).toEqual(response.INTERNAL_SERVER_ERROR);
   });
 });
+describe("getGuildRoles", () => {
+  it("should return undefined if status is not ok", async () => {
+    jest
+      .spyOn(global, "fetch")
+      .mockImplementationOnce(async () =>
+        Promise.resolve(new JSONResponse({}, { status: 500 }))
+      );
+
+    const response = await getGuildRoles(guildEnv);
+    expect(response).toBeUndefined();
+  });
+
+  it("should return array of objects containing role_id and role_name", async () => {
+    jest
+      .spyOn(global, "fetch")
+      .mockImplementationOnce(async () =>
+        Promise.resolve(new JSONResponse(guildDetailsMock))
+      );
+    const roles = await getGuildRoles(guildEnv);
+    const expectedRoles = [
+      {
+        id: "1234567889",
+        name: "@everyone",
+      },
+      {
+        id: "12344567",
+        name: "bot one",
+      },
+    ];
```

Comment:
Instead of creating a separate expectedRoles, please filter the roles from the mock data and return.
In the future, if there are any changes to the mock data in fixtures, you will not have to manually change it here.